alerts/cluster-monitoring-operator/TargetDown.md

```diff
@@ -55,13 +55,42 @@ We are ready to diagnose the root cause by inspecting each component on the scra
 workflow.
 
 ### Potential Issues and Resolutions
+
+#### Unknown Content Type
+
+Beginning with OpenShift Container Platform version 4.19, Prometheus enforces stricter
+rules on the `Content-Type` header when scraping.
+
+If the scraped target does not have a `Content-Type` header or if the header is
+unparsable or unrecognised and no [`fallback_scrape_protocol`] is set for the
+target, the scrape fails, and the `TargetDown` alert is fired.
+
+For more context, error logs similar to the following one should be present:
+
+   ```shell
+   # NAMESPACE='<value of namespace label from alert>'
+
+   $ oc -n $NAMESPACE logs -l 'app.kubernetes.io/name=prometheus' --tail=-1 | \
+   grep 'Failed to determine correct type of scrape target.*' \
+   | sort | uniq -c | sort -n
+   msg="Failed to determine correct type of scrape target." \
+   component="scrape manager" \
+   scrape_pool=<monitor>/<namespace>/<service-monitor-name>/<endpoint-id> \
+   target=concerned-target content_type="concerned-content_type" \
+   fallback_media_type="fallback_media_type-if-any" err="explicit_error"
+   ```
+
+In such a case, open a support case and attach the artifacts gathered
```

Comment:
I think we should make it clear what it is meant by artifacts or add link to diagnosis steps from openshift docs

Reply:
We (still) use that wording in many other places: https://github.com/search?q=repo%3Aopenshift%2Frunbooks%20artifacts&type=code
We can always make clearer in the future if needed, for now let's stay consistent.